core/common/src/main/java/alluxio/security/authorization/Mode.java

```diff
@@ -109,6 +109,8 @@ public Bits not() {
   private Bits mGroupBits;
   private Bits mOtherBits;
 
+  private Mode() {} // needed for JSON serialization and deserialization
```

Comment:
I review all the options object fields and this was the only field type that had this issue